### PR TITLE
DNS record Viewer - Python2

### LINKED ISSUE #65: [Feature] DNS Record Viewer
For a given hostname, fetch all the available DNS Records given below:

Record type | Record contains
-- | -- | 
'A' | IPv4 addresses (default) |
'AAAA' | IPv6 addresses | 
'ANY' | any records | 
'CNAME' | canonical name records | 
'MX' | mail exchange records | 
'NAPTR' | name authority pointer records | 
'NS' | name server records | 
'PTR' | pointer records | 
'SOA' | start of authority records |
'SRV' | service records | 
'TXT' | text records | 

> Additional task: Explain briefly about all the records in UI.

### PR DESCRIPTION
**Pull request by Kashish121**
### Description

Added a new feature : DNS record Viewer.
*The code is written in python2.*

Fixes #65 

Type of change: New feature (non-breaking change which adds functionality)

> This change requires a documentation update.
> Will add the steps and dependencies in the README.md once you(@vinitshahdeo ) review the code. 

To run:
``` python 
pip install dnspython
python src/dnsRecord.py
```

### CODE REVIEW

File: src/dnsRecord.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
+import socket 
+import dns
+import dns.resolver
+
```
Comment (vinitshahdeo) [inline src/dnsRecord.py:4]: @Kashish121 Extra new lines?
Is this recommended, not sure?

**Review by vinitshahdeo — COMMENTED** [review]
@Kashish121 Great job 👏 

Can you please update a table for different records in `README.md`?

**Comment by vinitshahdeo** [comment]
@ishika1727 Pull these changes and verify it from your end.

**Comment by ishika1727** [comment]
@vinitshahdeo @Kashish121 , I am getting this output when trying to search for postman.com

image

**Comment by Kashish121** [comment]
Will include this feature on `README.md` after the merge.
Thanks @ishika1727 for verification.

**Pull request merged**